### PR TITLE
feat: Add transport icons

### PR DESCRIPTION
**Pull request by ahtohbi4**
Add `bike`, `bus` and `car` icons and update `truck` icon

icon transport

### CODE REVIEW

**Review by locness3 — APPROVED** [review]

File: tags.json [hunk 1]
```diff
@@ -21,15 +21,18 @@
...
   "book-open": ["read", "library"],
   "book": ["read", "dictionary", "booklet", "magazine", "library"],
   "bookmark": ["read", "clip", "marker", "tag"],
   "box": ["cube"],
   "briefcase": ["work", "bag", "baggage", "folder"],
   "building": ["organisation"],
+  "bus": ["bus", "transport", "trip"],
   "calendar": ["date"],
   "camera": ["photo"],
+  "car": ["car", "transport", "trip"],
```
Comment (ericfennis) [inline tags.json:35]:
```suggestion
  "car": ["vehicle", "transport", "trip"],
```

File: tags.json [hunk 2]
```diff
@@ -21,15 +21,18 @@
...
   "bell-off": ["alarm", "notification", "silent"],
+  "bike": ["bike", "transport", "trip"],
   "bluetooth": ["wireless"],
   "book-open": ["read", "library"],
   "book": ["read", "dictionary", "booklet", "magazine", "library"],
   "bookmark": ["read", "clip", "marker", "tag"],
   "box": ["cube"],
   "briefcase": ["work", "bag", "baggage", "folder"],
   "building": ["organisation"],
+  "bus": ["bus", "transport", "trip"],
```
Comment (ericfennis) [inline tags.json:32]:
```suggestion
  "bus": ["bus", "vehicle", "transport", "trip"],
```

File: tags.json [hunk 3]
```diff
@@ -21,15 +21,18 @@
   "battery-charging": ["power", "electricity"],
   "bell": ["alarm", "notification", "sound"],
   "bell-off": ["alarm", "notification", "silent"],
+  "bike": ["bike", "transport", "trip"],
```
Comment (ericfennis) [inline tags.json:24]:
```suggestion
  "bike": ["bicycle", "transport", "trip"],
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Small additions

File: tags.json (hunk 2 above)
Comment (locness3) [reply tags.json]: Should remove the bus tag here qs well

File: icons/bus.svg [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M19 17h2l.64-2.54a6 6 0 000-2.92l-1.07-4.27A3 3 0 0017.66 5H4a2 2 0 00-2 2v10h2m10 0h-4"/>
```
Comment (Andreto) [inline icons/bus.svg:12]:
```suggestion
  <path d="M19 17h2l.64-2.54a6 6 0 000-2.92l-1.07-4.27A3 3 0 0017.66 5H4a2 2 0 00-2 2v10h2m10 0h-4" />
```

File: icons/car.svg [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14 16H9m10 0h3v-3.15a1 1 0 00-.84-.99L16 11l-2.7-3.6a1 1 0 00-.8-.4H5.24a2 2 0 00-1.8 1.1l-.8 1.63A6 6 0 002 12.42V16h2"/>
```
Comment (Andreto) [inline icons/car.svg:12]:
```suggestion
  <path d="M14 16H9m10 0h3v-3.15a1 1 0 00-.84-.99L16 11l-2.7-3.6a1 1 0 00-.8-.4H5.24a2 2 0 00-1.8 1.1l-.8 1.63A6 6 0 002 12.42V16h2" />
```

File: icons/truck.svg [hunk 6]
```diff
@@ -9,8 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="1" y="3" width="15" height="13" />
-  <polygon points="16 8 20 8 23 11 23 16 16 16 16 8" />
-  <circle cx="5.5" cy="18.5" r="2.5" />
-  <circle cx="18.5" cy="18.5" r="2.5" />
+  <path d="M10 17h4V5H2v12h3M20 17h2v-3.34a4 4 0 00-1.17-2.83L19 9h-5v8h1"/>
```
Comment (Andreto) [inline icons/truck.svg:12]:
```suggestion
  <path d="M10 17h4V5H2v12h3M20 17h2v-3.34a4 4 0 00-1.17-2.83L19 9h-5v8h1" />
```

**Review by ericfennis — APPROVED** [review]

**Pull request merged**